apps/server/src/tasks/sendTestEmail.ts

```diff
@@ -1,18 +0,0 @@
-import { makeDonation, makeFundraiser, makePayment } from '../../local/testHelpers';
-import { sendEmail } from '../helpers/email';
-import confirmation from '../helpers/email/confirmation';
-
-const testEmail = 'raisedemo@gmail.com';
-
-export default {
```

Comment:
It might be useful to keep this file around, for more easy debugging

Reply:
This task should still be here, just under a different name ( sendtestconfirmationemail.ts ), as I thought it may prevent people from using the wrong task. Thought I might as well change the file name to match the actual name for ease in the future